databricks/koalas/internal.py

```diff
@@ -77,6 +78,8 @@
 DEFAULT_SERIES_NAME = 0
 SPARK_DEFAULT_SERIES_NAME = str(DEFAULT_SERIES_NAME)
 
+dtype_data_types = (np.dtype, ExtensionDtype)
```

Comment:
Shall we use the same way as https://github.com/databricks/koalas/blob/master/databricks/koalas/typedef/typehints.py#L497-L502 with `__union_params__` instead of `__tuple_params__`?

Reply:
And shall we maybe put this right after the definition of `Dtype`?

Reply:
Sounds good! `__union_params__` is used, and it is put right after the definition of `Dtype`.